src/helper functions/dateFormat.js

```diff
@@ -0,0 +1,7 @@
+var formatDate = (date) => {
```

Comment:
dayjs has utility for formatting dates. You didn't have to do this.

- https://day.js.org/docs/en/display/format